### PR TITLE
docs: Update CSI-RX docs for AM62, AM62P and AM62A

### PR DESCRIPTION
**Pull request by Rishikesh-D**
1. Add table to list down latest SDK supported sensors and daughter boards
2. Add command to apply overlay for OV2312 connected to the DS90UB954-Q1 EVM
3. Add documentation for OV5640 + Arducam V3Link (Fusion Mini) support on SK-AM62P

### CODE REVIEW

File: source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst [hunk 1]
```diff
@@ -670,6 +726,29 @@ Enabling camera sensors
...
 
+    CSI2RX testing details
+    ======================
+
+    Following sensors and daughter cards have been tested with the latest SDK.
+
+        +--------------+---------------------------------------+-----------------------------------+
+        | Hardware     | Sensor                                | Default format and resolution     |
+        +--------------+---------------------------------------+-----------------------------------+
+        | SK-AM62A     | IMX219 RPi Camera                     | SRGGB8_1X8/1920x1080 at 30 fps    |
```
Comment (devarsht) [inline source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst:737]: As discussed let's have separate columns for media bus format, video format and resolutions that we have validated. Also use standard macro names (MEDIA_BUS_FMT and V4L2_PIX_FMT) while specifying them on table

**Review by cshilwant — CHANGES_REQUESTED** [review]
https://github.com/TexasInstruments/processor-sdk-doc/pull/91#discussion_r1879416197

**Review by devarsht — APPROVED** [review]

File: source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst [hunk 2]
```diff
@@ -220,6 +220,12 @@ Enabling camera sensors
     |__PART_FAMILY_NAME__| SK supports the following 15-pin FFC compatible
     camera modules with **OV5640** sensor:
 
+.. ifconfig:: CONFIG_part_variant in ('AM62PX')
+
+    Arducam V3Link (Fusion Mini) board can also be used to connect these camera modules to SK-AM62P over FPDLink cable.
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst:225]: For all ifconfigs & code-block directives in this file ensure that it clearly lines up with directive.

Refer - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

File: source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst [hunk 3]
```diff
@@ -435,6 +455,43 @@ Enabling camera sensors
...
     :ref:`Users Guide <users-guide-kernel-config>`.
 
+.. ifconfig:: CONFIG_part_variant in ('AM62X')
+
+    CSI2RX testing details
+    ======================
+
+    Following sensors have been tested with the latest SDK.
+
+        +------------------------------------+----------------------------+----------------------+------------+
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst:465]: As you have 2 tables back to back, I would encourage you to use csv-table - https://docutils.sourceforge.io/docs/ref/rst/directives.html#csv-table-1

It allows use to provide table names. In addition each table gets a unique ID. So, you can directly have a link to it.
Eg - https://software-dl.ti.com/processor-sdk-linux/esd/AM62X/10_00_07_04/exports/docs/devices/AM62X/linux/Linux_Performance_Guide.html#id6 points directly to Dhrystone Benchmarks

Refer - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/source/devices/AM62X/linux/Linux_Performance_Guide.rst?plain=1#L67 to check how @praneethbajjuri uses it in `Linux_Performance_Guide`

File: source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst [hunk 4]
```diff
@@ -670,6 +731,33 @@ Enabling camera sensors
         sink_1::startx="<640>" sink_1::starty="<480>" sink_1::widths="<640>" sink_1::heights="<480>" ! \
         queue ! kmssink driver-name=tidss plane-properties=s,zpos=1
 
+    CSI2RX testing details
+    ======================
+
+    Following sensors and daughter cards have been tested with the latest SDK.
+
+        +------------------------------------+----------------------------+----------------------+------------+
```
Comment (cshilwant) [inline source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst:739]: Same recommendation of using csv-table as above

**Review by cshilwant — DISMISSED** [review]

File: source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst (hunk 4 above)
Comment (Rishikesh-D) [reply source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst]: Hello @cshilwant, I have done the requested changes, please review.

**Review by devarsht — APPROVED** [review]

**Review by StaticRocket — CHANGES_REQUESTED** [review]

File: source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst [hunk 5]
```diff
@@ -137,755 +137,831 @@ I2C2 and CSI-RX Port 0:
...
+   .. code-block:: console
 
-    .. code-block:: console
+      $ cam -c1 --stream width=640,height=480,pixelformat=UYVY -C20 -F#.uyvy
+      $ ls *.uyvy
+     -rw-r--r-- 1 root root 614400 Jan  1 19:19 cam0-stream0-000000.uyvy
+     -rw-r--r-- 1 root root 614400 Jan  1 19:19 cam0-stream0-000001.uyvy
+     -rw-r--r-- 1 root root 614400 Jan  1 19:19 cam0-stream0-000002.uyvy
+     -rw-r--r-- 1 root root 614400 Jan  1 19:19 cam0-stream0-000003.uyvy
+     -rw-r--r-- 1 root root 614400 Jan  1 19:19 cam0-stream0-000004.uyvy
```
[the remark this answers is not in the document]
Comment (Rishikesh-D) [reply source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst]: Thank you for the pointer. Fixed it.

**Review by StaticRocket — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Pull request merged**